x-pack/legacy/plugins/spaces/public/views/management/edit_space/enabled_features/feature_table.tsx

```diff
@@ -102,9 +102,7 @@ export class FeatureTable extends Component<Props, {}> {
             id={record.feature.id}
             checked={checked}
             onChange={this.onChange(record.feature.id) as any}
-            aria-label={
-              checked ? `${record.feature.name} visible` : `${record.feature.name} disabled`
-            }
+            label={checked ? `${record.feature.name} visible` : `${record.feature.name} disabled`}
```

Comment:
The change from `aria-label` to `label` is adding some visual noise to the Spaces management screen that I don't think is ideal:

![image](https://user-images.githubusercontent.com/3493255/67963638-538f0b00-fbd5-11e9-88d0-0cbcc9a9ddb2.png)


Is there another way to convey this information? Is `label` a hard requirement for using `EuiSwitch` now?

Reply:
Ah! I'll get this updated. There is now a `showLabel` prop that I somehow forgot to add here. Setting it to `false` will render label-less switches. `label` gets used as `aria-label` when this option is set.

Reply:
Perfect, thanks @thompsongl!

Reply:
After adding `showLabel={false}`:

<img width="1150" alt="Screen Shot 2019-10-31 at 11 30 48 AM" src="https://user-images.githubusercontent.com/2728212/67966461-e8900500-fbd1-11e9-8e36-50fde16c178e.png">

Commit incoming.